src/main/java/com/realestate/service/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.realestate.service.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+@EnableWebSecurity
```

Comment:
```suggestion
@EnableWebSecurity
```

```@EnableWebSecurity``` 어노테이션은 내부에 ```@Configuration``` 어노테이션이 포함되어있어서 
생략해도 될 거 같아요:)